### PR TITLE
NW6 | Rabia Avci | Full-Stack-Project | Week-1 | List video recommendations

### PR DESCRIPTION
**Pull request by RbAvci**
Completed these tasks for this ticket:

 On page load, the website should display a list of video recommendations

 For each video recommendation:

- It shows the video's title
- If you click on the title, it redirects you to the YouTube page for that video
- add a simple CSS file for this component

### CODE REVIEW

File: client/src/VideoRecommendations.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	}, []);
+
+	return (
+		<div className="video-list">
+			{videos.map((videoData, i) => (
+				<div className="video" data-testid="video" key={i}>
+					<div>
+						{/* <a>{videoData.title}</a> */}
+						<a href={videoData.src}>{videoData.title}</a>
+					</div>
```
Comment (Ara225) [inline client/src/VideoRecommendations.jsx:31]: Good work, no complaints!

File: client/src/VideoRecommendations.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+:root {
+	max-width: 95%;
+	margin: auto;
+}
+
+.video-list {
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
```
Comment (Ara225) [inline client/src/VideoRecommendations.css:8]: Bit of responsive design, good to see :)

File: client/src/VideoRecommendations.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	gap: 20px;
+}
+
+.video {
+	border: 1px solid #ccc;
+	border-radius: 10px;
```
Comment (Ara225) [inline client/src/VideoRecommendations.css:14]: Nice touch

File: client/src/VideoRecommendations.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+	padding: 20px;
+	background-color: #f9f9f9;
+	font-family: Arial, sans-serif;
+}
+
+a {
+	color: #007bff;
+	text-decoration: none;
```
Comment (Ara225) [inline client/src/VideoRecommendations.css:22]: This is bad from a UI POV - links are usually underlined and blue do if you change both, it's possible the user might not realuze it's a link

**Review by Ara225 — COMMENTED** [review]
One UI comment, other than that good wor. Love how detailed your pull request descriptions are BTW

File: client/src/VideoRecommendations.css (hunk 4 above)
Comment (RbAvci) [reply client/src/VideoRecommendations.css:22]: oh, thank you, done!

**Pull request merged**